.github/workflows/validate.yml

```
@@ -0,0 +1,16 @@
+name: Validate
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

May as well start on latest version
```suggestion
        uses: actions/checkout@v3
```

---

ah yes.